src/test/java/BuildResponseTest.java

```diff
@@ -0,0 +1,46 @@
+import http.response.BuildResponse;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BuildResponseTest {
+    @Test
+    public void checkThatStartLineIsReturned (){
+        BuildResponse build = new BuildResponse();
+        int statusCode = 200;
+        String statusMessage = "OK";
+        String protocol = "HTTP/1.1";
+
+        assertArrayEquals(build.buildStartLine(protocol,statusCode,statusMessage),"HTTP/1.1 200 OK\n\r".getBytes());
+
+
+    }
+
+    @Test
+    public void checkThatHeadersLineIsParsed (){
+        BuildResponse build = new BuildResponse();
+        String headerKey = "Age";
+        String headerValue = "7777";
+
+        assertEquals(build.parseHeaderLine(headerKey,headerValue),"Age : 7777\n\r");
+
```

Comment:
I'm by no means Java expert and I'm aware that Java syntax requires you to add empty lines in places that are untypical for other languages like Python, ruby or JS. 

But the indentation in your file is not consistent - sometimes there are two empty lines between `assert` and `}`, sometimes one, make sure it's all aligned according to one standard.

Reply:
I'll check the indentation, InteliJ does the formatting for me but i'll double check.